lib/inspec/base_cli.rb

```diff
@@ -370,15 +370,15 @@ def config
     # get the log level
     # DEBUG < INFO < WARN < ERROR < FATAL < UNKNOWN
     def get_log_level(level)
-      valid = %w{debug info warn error fatal}
+      valid = %w{trace debug info warn error fatal}
 
       if valid.include?(level)
         l = level
       else
         l = "info"
       end
 
-      Logger.const_get(l.upcase)
+      Mixlib::Log.const_get(l.upcase)
```

Comment:
here we change the logger library itself .. is this necessary ? are we sure it has not caused any formatting changes or any changes to output and available methods across Inspec?

Reply:
No this will not cause any formatting changes - This is only used to access the value - Logger library itself does not support `trace`. In mixlib log there is a monkey patch done on logger for extending trace

Reply:
This method only does get operation of constant